packages/compass-shell/src/components/shell-header/shell-header.jsx

```diff
@@ -15,6 +15,20 @@ export class ShellHeader extends Component {
     showInfoModal: PropTypes.func.isRequired
   };
 
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleKeyboardToggle.bind(this));
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.handleKeyboardToggle.bind(this));
+  }
+
+  handleKeyboardToggle({ ctrlKey, keyCode }) {
```

Comment:
Just FYI `keyCode` [is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)

Reply:
I'll create a follow up PR

Reply:
[Fixed here](https://github.com/mongodb-js/compass/pull/2660)